modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaGetPortletGroupIdMethodCheck.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaGetPortletGroupIdMethodCheck extends BaseFileCheck {
+
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		if (!fileName.endsWith(".java") && !fileName.endsWith(".jsp") &&
+			!fileName.endsWith(".jspf") && !fileName.endsWith(".ftl")) {
```

Comment:
Currently, upgrade checks are not able to process `.jsp`, `.jspf` and `.ftl` files. You will need to update `UpgradeSourceProcessor.java` to include checking those filetypes.

Reply:
Check here: <https://github.com/kevhlee/liferay-portal/blob/master/modules/util/source-formatter/src/main/java/com/liferay/source/formatter/processor/UpgradeSourceProcessor.java#L36-L38>

Reply:
done.